### ISSUE TITLE
Cluster create don't work since yesterday

### ISSUE THREAD

**Issue by smirnov-mi** [xetys/hetzner-kube#236]
ERROR MESSAGE:

run failed
command:apt-get update && apt-get install -y apt-transport-https ca-certificates curl software-properties-common
stdout:Reading package lists...

stderr:Process exited with status 100

COMMAND RUNNING:
hetzner-kube -d cluster create -d --name MMM --ssh-key demo --master-server-type cx11 --worker-server-type cx11 --ha-enabled true -m 3 -w 3 --self-hosted true  --datacenters nbg1-dc3,hel1-dc2

TESTED WITH:  v0.3.1 and v0.3.2 Linux amd64

**Comment by Exellion** [comment]
I have the same problem

**Comment by Quantower** [comment]
Yes, does not work for me too.

**Comment by Exellion** [comment]
> FYI: Just rebuilt hetzner-kube from sources (master branch), it's working.

What version of GO do you use? I try with 1.11, but errors still occured

**Comment by smirnov-mi** [comment]
go get -u github.com/xetys/hetzner-kube
go version go1.10.4 linux/amd64

**Comment by mavimo** [comment]
maybe related to https://github.com/xetys/hetzner-kube/issues/237 ?

**Comment by smirnov-mi** [comment]
nope, it was different. BTW. rebuild the hetzner-kube again. Just tested the older version and it didn't work (2.Jan.2019). Rebuild from the current sources - OK.

**Comment by Quantower** [comment]
We see this problem again. Rebuilding of hetzner-kube does not help

**Comment by tckb** [comment]
same issue for me too. For me, it says ```parse key failed:ssh: cannot decode encrypted private keysparse key failed:ssh: cannot decode encrypted private keysparse key failed:ssh: cannot decode encrypted private keys2019/10/21 15:44:14 parse key failed:ssh: cannot decode encrypted private keys``` latest version from master branch. 

anyone has a fork that works?

**Comment by mavimo** [comment]
@tckb which key are you using RSA DSA ECDSA?

**Comment by tckb** [comment]
@mavimo I tried with RSA and ECDSA both failed. It seems to work with only password-less key

**Comment by xetys** [comment]
still present?

**Comment by tckb** [comment]
@xetys last time I created my cluster it still didn't work. I am currently working with password-less key for my cluster

**Comment by xetys** [comment]
When was the last time? Still present with v0.5.0 ?

**Comment by tckb** [comment]
I believe this was the latest version, I used the latest version from master branch I think.

**Comment by xetys** [comment]
Very strange... Do you use any specific kind of SSH keys? Operation system?

**Comment by tckb** [comment]
@xetys nothing special. Although, as I mentioned, the version that I used is a little bit old. I need to check and update the version with the one in the master. I am currently on macOS
